### PR TITLE
run rustfmt on librustc_metadata library

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
cc @nrc

File: src/librustc_metadata/astencode.rs [hunk 1]
```diff
@@ -445,40 +449,37 @@ fn encode_method_callee<'a, 'tcx>(ecx: &e::EncodeContext<'a, 'tcx>,
...
-                }).unwrap(),
-                substs: this.read_struct_field("substs", 3, |this| {
-                    Ok(dcx.tcx.mk_substs(this.read_substs(dcx)))
-                }).unwrap()
-            }))
-        }).unwrap()
+                let autoderef = this.read_struct_field("autoderef", 0, Decodable::decode)
+                                    .unwrap();
+                Ok((autoderef,
+                    ty::MethodCallee {
```
Comment (nrc) [inline src/librustc_metadata/astencode.rs]: https://github.com/rust-lang-nursery/rustfmt/issues/1019

Comment (bluss) [inline src/librustc_metadata/astencode.rs]: The closure body of `read_struct` is twice indented, is that intentional?

Comment (marcusklaas) [inline src/librustc_metadata/astencode.rs]: I'd say no, but opinions diverge on this subject.

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to reopen with a rebase!

**Pull request closed without merging**